wp-content/themes/core/blocks/tribe/copyright/edit.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * React hook that is used to mark the block wrapper element.
+ * It provides all the necessary props like the class name.
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-block-editor/#useblockprops
+ */
+import { useBlockProps, InspectorControls } from '@wordpress/block-editor';
+import { PanelBody, TextControl, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * The edit function describes the structure of your block in the context of the
+ * editor. This represents what the editor will render when the block is used.
+ *
+ * @ignore
+ *
+ * @see https://developer.wordpress.org/block-editor/reference-guides/block-api/block-edit-save/#edit
+ *
+ * @return {Element} Element to render.
+ */
```

Comment:
❓ Idk how you feel about these default comments, but I usually like to remove them and replace them with my own if possible. If you feel they are helpful to other devs as-is, I'm fine with leaving them.

Reply:
I've updated these. Just pointing out that the Tribe Terms block is also still using the defaults. We many need a backlog ticket to do a pass on cleaning up generic comments?